plugins/inputs/procstat/procstat.go

```diff
@@ -391,6 +457,62 @@ func (p *Procstat) SimpleFindPids(f PIDFinder) ([]PID, map[string]string, error)
 // execCommand is so tests can mock out exec.Command usage.
 var execCommand = exec.Command
 
+type MainStatus struct {
+	Map map[string]string
+}
+
+func BytesToString(b []byte) string {
+	return unsafe.String(&b[0], len(b))
+}
+
+func (p *Procstat) supervisorPIDs() ([]string, map[string]MainStatus, error) {
+	var Services string
+	var servicelist []string
+
+	if p.SupervisorUnit != "" {
+		servicelist = strings.Split(p.SupervisorUnit, ",")
+		Services = strings.Join(servicelist, " ")
+	}
+
+	out, err := execCommand("supervisorctl", "status", Services).Output()
```

Comment:
Did you check that this command really exists in `Init()`?

Reply:
I'm not sure how to check. If this command does not exist in the environment, it should not have an impact, right?